pkg/kubernetes/views/imagestream-body.html

```diff
@@ -1,3 +1,16 @@
+<div ng-repeat="statustags in stream.status.tags">
+  <div ng-repeat="condition in statustags.conditions" ng-if='condition.status == "False"'
```

Comment:
I think you're right. I confirmed this based on the current types: https://github.com/openshift/origin/blob/master/pkg/image/api/v1/types.go#L139